workflow_test.go

```diff
@@ -0,0 +1,187 @@
+package workflow_test
+
+import (
+	"fmt"
+
+	"github.com/gol4ng/workflow"
+)
+
+type MyObject struct {
+	*workflow.Marking
```

Comment:
I'm still not OK with that coupling 😭

Reply:
- You still not give an alternative instead of crying..
- You still not read the code!!!

i advise you to read https://github.com/gol4ng/workflow/pull/1/files#diff-def19e92a3bab12d9d1c3ac803fb7accR7
that allow you to implement your own marking storer and let you store marking information outside of subject if you want